pkg/execute/executor.go

```diff
@@ -169,6 +169,14 @@ func (e *DefaultExecutor) Execute(ctx context.Context) interactive.Message {
 	}
 
 	if isPluginCmd {
+		if isHelpCmd(cmdCtx.Args) {
+			msg, err := e.pluginExecutor.Help(ctx, e.conversation.ExecutorBindings, cmdCtx.Args, cmdCtx.CleanCmd)
+			if err != nil {
+				e.log.Errorf("while executing help command %q: %s", cmdCtx.CleanCmd, err.Error())
+				return empty
+			}
+			return msg
+		}
```

Comment:
1. We should report the help command usage as well
2. Once you add the command reporting, then I think the code can look even more complicated - consider extracting this help usage to a separate small method.